src/test/java/tracko/logic/commands/CommandTestUtil.java

```diff
@@ -2,6 +2,7 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 // import static org.junit.jupiter.api.Assertions.assertTrue;
```

Comment:
I think it would be would to remove the duplicated import statements that are commented out! Do look out for these in the other files as well

Reply:
Nice catch! Will remove before merging 👍